src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -4302,20 +4301,54 @@ def test_single_stencil_literal(dist_mem, tmpdir):
     assert output6 in result
 
 
-def test_stencil_region_unsupported(dist_mem):
-    '''Check that we raise an exception if the value of the stencil type
-    in stencil(<type>[,<extent>]) is region. This is not a parse error
-    as region is a valid value, it is just that the LFRic
-    infrastructure does not yet support it. '''
+def test_stencil_region(dist_mem, tmpdir):
```

Comment:
Would you mind creating a new test file, `tests/domain/lfric/lfric_stencil_tests.py` and moving at least this test there. If you're feeling really generous you might move the other stencil tests too :-)

Reply:
I've moved everything I believe to be a mainly stencil test to a new file, this doesn't include tests of things like halo exchange tests that are testing for stencil compatibility for instance. See commit 3306ebb8ff4b9499282e0f3adac6305b55b0612f

Reply:
Wow, there was a lot! Thanks very much.